core/domain/feedback_domain.py

```diff
@@ -37,9 +56,20 @@ def __init__(self, full_thread_id, exploration_id, state_name,
         self.last_updated = last_updated
 
     def get_thread_id(self):
+        """
+        
```

Comment:
@Andimeo - Do we just want the blank line before the attributes defined for the class in the docstring? And I can remove the blank line for the method-level return docstrings? I guess @seanlip was only talking about the attributes. I can fix that.